app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -498,6 +567,32 @@ private void updateTheDarkness() {
         image.setAlpha(1.0f - scrollPercentage);
     }
 
+    /**
+    * Returns captions for media details
+     *
+     * @param media object of class media
+     * @return caption as string
+     */
+
+    private String prettyCaption(Media media) {
+        String caption = media.getCaption().trim();
+        if (caption.equals("")) {
+            return getString(R.string.detail_caption_empty);
+        } else {
+            return caption;
+        }
+    }
+
+    private ArrayList<Map<String, String>> getDepictions(Media media) {
```

Comment:
javadoc, including description of key/value

Reply:
there was no usage of this function. Hence removed